lib/constants.go

```diff
@@ -986,7 +992,8 @@ var TestnetForkHeights = ForkHeights{
 	DeSoUnlimitedDerivedKeysBlockHeight: uint32(467217),
 
 	// FIXME: Set to real block height when we're ready.
-	AssociationsBlockHeight: math.MaxUint32,
+	AssociationsBlockHeight:          math.MaxUint32,
+	AllowUpdatingNFTPostsBlockHeight: math.MaxUint32,
```

Comment:
we'll combine these block heights - but that can happen separately.

Reply:
Resolved that in a subsequent PR: https://github.com/deso-protocol/core/pull/451.